index.html

```diff
@@ -180,6 +180,22 @@ <h3>Skills</h3>
             </ul>
         </section>
     </section>
+    <section class="last">
+        <div class="footer">
+        <h1 class="contact-title">Contact me</h1>
+        <p class="contact-description">If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it </p>
+        <br>
+        <form action="https://formspree.io/f/mvolzwlj" method="POST" class="contact-form">
+            <input type="text" name="Name" class="area name" placeholder="Name" maxlength="30" required>
+            <br>
+            <input type="email" name="email" class="area email" placeholder="Email" required>
+            <br>
+            <textarea name="message" class="message" maxlength="500" placeholder="Write your message here" required></textarea>
+            <br>
+        </form>
+        </div>
+        <button type="submit" class="submit">Get in touch</button>
```

Comment:
- [ ] This button is not working.  The reason is that it is outside your `<form>` tag.  

Please kindly just move the `<button type="submit" class="submit">Get in touch</button>` line to be inside your form tag. 

![image](https://user-images.githubusercontent.com/57421823/150558450-b0a4eec9-ff60-46b5-a79b-e556d59462d3.png)

That way, your button will start working 💪